packages/code_excerpt_updater/mono_pkg.yaml

```diff
@@ -1,6 +1,6 @@
 # See https://github.com/google/mono_repo.dart for details on this file
 sdk:
-- 2.15.0
+- 2.16.0
```

Comment:
Might want to bump the min SDK of the corresponding pubspecs, too! (here and below)